### PR TITLE
Feature/add title

### PR DESCRIPTION
**Pull request by jemalSparklingWater**
- Created and styled categories section

### CODE REVIEW

File: oasis/src/App.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.App {
+  min-width: 600px;
+}
```
Comment (gitFaisal) [inline oasis/src/App.css]: Having a hardcoded min-width of 600px might be the reason our app doesn't adjust to phone screen size. Look into how we can improve this as a homework.

File: oasis/src/App.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  width: 100%;
+  height: 30rem;
+  background-size: cover;
+}
+
+#title {
+  font-size: 180px;
+  color: white;
+  text-shadow: 2px 2px 4px rgb(0, 0, 0);
+}
```
Comment (gitFaisal) [inline oasis/src/App.css]: The title font is hardcoded, which means on smaller screens the text will be way too large. Look into how we can dynamically change the size of the text to fit different screen sizes.

File: oasis/src/components/Categories/CategoryStyles.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+.horizontal-scroll li:last-child {
+  border-right: none;
+}
+
+.horizontal-scroll a {
+  text-decoration: none;
+  color: black;
+  display: inline-block;
+  flex: 1; /* t items will make them grow or shrink to fill unused*/
```
Comment (gitFaisal) [inline oasis/src/components/Categories/CategoryStyles.css]: Remove comment

File: oasis/src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import logo from "../../images/logo-cropped.png";
+import "./NavbarStyles.css";
+import Categories from "../Categories/Categories";
+
+const Navbar = () => {
+  return (
+    <div>
+      {/* Original Navbar code */}
```
Comment (gitFaisal) [inline oasis/src/components/Navbar/Navbar.jsx]: remove comment

**Review by gitFaisal — CHANGES_REQUESTED** [review]
Make the changes requested and we should be good to merge!

File: oasis/src/App.css (hunk 1 above)
Comment (jemalSparklingWater) [reply oasis/src/App.css]: 👍

File: oasis/src/App.css (hunk 2 above)
Comment (jemalSparklingWater) [reply oasis/src/App.css]: 👍

File: oasis/src/components/Categories/CategoryStyles.css (hunk 3 above)
Comment (jemalSparklingWater) [reply oasis/src/components/Categories/CategoryStyles.css]: 👍

File: oasis/src/components/Navbar/Navbar.jsx (hunk 4 above)
Comment (jemalSparklingWater) [reply oasis/src/components/Navbar/Navbar.jsx]: 👍

**Review by gitFaisal — APPROVED** [review]

**Pull request merged**